app/views/webhooks/index.html.erb

```diff
@@ -78,4 +85,54 @@
     </table>
   <% end %>
   <h2></h2>
+
+  <h1>Outbound Webhooks</h1> 
+  <p>Outbound webhooks POST to an external url after a deploy has finished. The request body will contain deploy, stage, an project information.</p>
+  <% if @project.deploys.any? %>
+    <p>Here is an example payload of this project's latest deploy:</p>
+    <p><%= DeploySerializer.new(@project.deploys.last).as_json %></p>
```

Comment:
won't this be pretty big ? ... hide it and toggle via a link or render in a new action ?